### PR TITLE
Adding notes for wwdc22 "Whats new in AppKit"

### PR DESCRIPTION
**Pull request by dagronf**
Added notes for the "Whats new in AppKit" session 10074 WWDC22

### CODE REVIEW

File: content/notes/wwdc22/10074.md [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+1. [Stage Manager](#stage-manager)
+2. [Preferences](#system-preferences)
+3. [Controls](#controls)
+	1. [NSComboButton](#nscombobutton)
+	2. [NSColorWell](#nscolorwell)
+	3. [NSToolBar](#nstoolbar)
+	4. [NSAlert](#nsalert)
+	5. [NSTableView](#nstableview)
+4. [SF Symbols](#sf-symbols)
+5. [Sharing](#sharing)
```
Comment (zntfdr) [inline content/notes/wwdc22/10074.md:18]: @wwdcNotes doesn't support this yet, I will remove it for now 🙏🏻

File: content/notes/wwdc22/10074.md [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+* Adopt the new styles when you're ready, or not at all.
+* New interface style for all configuration options with a new design language. 
+* Best way to adopt the new style is with SwiftUI.
+
+### What about AppKit?
+
+* Suggests incremental adoption of SwiftUI into your existing app.
+* Note - no indication of how to do this in AppKit directly.
+* See ["SwiftUI with AppKit"](https://developer.apple.com/wwdc22/10075) WWDC22
+
```
Comment (zntfdr) [inline content/notes/wwdc22/10074.md:52]: Wow! Apple pushing hard on SwiftUI in an AppKit session 😮

Comment (dagronf) [reply content/notes/wwdc22/10074.md:52]: Yeah it seems like they're drawing the line a bit here.  My guess is that large UI changes (eg. here's a new preferences framework) will be SwiftUI, whereas smaller items that need consistency (eg. here's a new control!) will be lower level AppKit. Or SwiftUI elements that are just wrappers around an AppKit control (eg. NSColorWell).

Thankfully they've make it quite easy (easier than for iOS which is strange!) to embed SwiftUI in an Appkit View.

**Comment by zntfdr** [comment]
Fantastic note! Thank you @dagronf 😃
[The note is now live](https://wwdcnotes.com/notes/wwdc22/10074/)

**Pull request merged**